### DISCUSSION TITLE
黑神话开服

### DISCUSSION THREAD

**Discussion by xian-niao** [share121/inter-knot#3237] [其他]
![e47ca14a330b25e6903f61d7cab21a0](https://github.com/user-attachments/assets/e2c9e720-ba2b-4954-96e4-eb30c18b6dd8)
黑神话悟空马上开服了，虽然说的上午10点，但游戏科学一般会提前偷跑，所以建议7点就打开游戏等着，可以抢一个靠前的uid。进入游戏之后，肝一下序章，因为剧情不能跳过，大概要4个小时。这时候打开邮箱，领取开服送的20抽福利，能有50抽，建议直接去抽新手池，8折，大奖孙悟空，目前最强主C，比天命人好用。如果抽到了孙悟空，建议直接补专武如意金箍棒，命座就不要想了，平民可以抽个2命，氪佬6+5随意。如果运气不好歪了常驻，常驻里面只有小钻风有点用，其他建议重开。

**Comment by TetsuyaTetsuya** [comment]
我嘞个猿神啊

**Comment by Nie7bai** [comment]
真猿神启动了啊！
